notebooks/en/agent_rag.ipynb

```diff
@@ -0,0 +1,2800 @@
+{
```

Comment:
<p>why not use Llama 3? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/135/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
Thank you, I added a more complete evaluation part with the comparison of agentic vs standard RAG (scores goes from 70% to 78.5%! 🚀)